samples/SendSample/Program.cs

```diff
@@ -46,7 +39,7 @@ public static void Main(string[] args)
                 try
                 {
                     // Create a new brokered message to send to the queue
-                    var message = new BrokeredMessage($"Message {i}");
+                    var message = new Message($"Message {i}");
```

Comment:
BrokeredMessage is an internal concept which doesn't need to be used publicly. Hence the reason we are changing it. I agree there might be a small number of cases when the word Message might be used already, but that might not be the majority scenario. And even in this case, the issue can be resolved using namespace prefix. Since this is a new API, we plan to have the nomenclature crisp and avoid all the internal details